src/transformer/mockDefiner/mockDefiner.ts

```diff
@@ -36,19 +39,31 @@ export class MockDefiner {
         return this._instance;
     }
 
-    public currentTsAutoMockImportName: ts.Identifier;
-
     public setFileNameFromNode(node: ts.TypeNode): void {
-        // tslint:disable-next-line:no-any
-        const thisFile: any = node.getSourceFile();
+        const thisFile: ts.SourceFile = node.getSourceFile();
         this._fileName = thisFile.fileName;
     }
 
     public setTsAutoMockImportIdentifier(): void {
         if (!this._neededImportIdentifierPerFile[this._fileName]) {
-            this._neededImportIdentifierPerFile[this._fileName] = ts.createFileLevelUniqueName(`${urlSlug(this._fileName, '_')}_repository`);
+            this._neededImportIdentifierPerFile[this._fileName] = Object.keys(ModulesImportUrl).map((key: ModuleName) => {
+                return {
+                    name: key,
+                    moduleUrl: ModulesImportUrl[key],
+                    identifier: this._createUniqueFileName(key),
+                };
+            });
         }
-        this.currentTsAutoMockImportName = this._neededImportIdentifierPerFile[this._fileName];
+    }
+
+    private _getModuleIdentifier(fileName: string, module: ModuleName): ts.Identifier {
```

Comment:
Can you move this private method at the bottom?

Reply:
NOOOOOOOOO!!.. Of course :)